src/main/java/org/launchcode/techjobs/mvc/controllers/SearchController.java

```diff
@@ -23,4 +29,19 @@ public String search(Model model) {
 
     // TODO #3 - Create a handler to process a search request and render the updated search view.
 
+    @RequestMapping(value = "results")
```

Comment:
```suggestion
    @PostMapping(value = "results")
```
The functionality of your `displaySearchResults` method works as expected, but the assignment recommends that you use a different annotation instead of `@RequestMapping` based on the value `method` in the `form` element of `search.html`: https://education.launchcode.org/java-web-development/assignments/tech-jobs-mvc.html#complete-searchcontroller. Once this is updated to the annotation it is expecting, the test associated with it will pass!